index.js

```diff
@@ -0,0 +1,7 @@
+'use strict'
+
+if (process.env.NODE_ENV === 'development') {
+  module.exports = require('./dist/dio.cjs.development.min.js')
```

Comment:
I missed this before but the file name seems wrong, should be `dio.umd.development.min.js`. The `dio.cjs.development.min.js` entry is not a main entry point but the node server entry point. I guess the tests for `propTypes` will also serve to catch this.

Reply:
I've changed this accordingly in branch "issue76":
https://github.com/thysultan/dio.js/commit/99dad6ec07b353521e36cc26bd19d11b1ccdada1#diff-168726dbe96b3ce427e7fedce31bb0bc